pkg/web/templates/transactions_accept.html

```diff
@@ -67,6 +67,9 @@ <h1 class="font-semibold text-2xl md:text-3xl text-center text-balance">Accept T
       }
     }
 
+    // Convert the transaction amount to a float.
+    data.transaction.amount = parseFloat(data.transaction.amount);
```

Comment:
What happens if this errors? Does it just set the amount to empty string?

Reply:
A great question. If the amount can't be parsed, `NaN` will be returned. The user will see a data parse error in the UI. I'll update this to handle the conversion better.